lib/uplink/bucket.go

```diff
@@ -19,8 +19,9 @@ import (
 // Bucket represents operations you can perform on a bucket
 type Bucket struct {
 	BucketConfig
-	Name    string
-	Created time.Time
+	Name        string
+	Attribution bool
```

Comment:
should be a string?

Reply:
Added **Attribution** field  for two purposes (1)to test the  bucket attribution during open bucket operations(). We don't exit if attribution is false, but just log the info and continue.So to test I need a variable to verify. Also (2) thought having a separate field that identifies the bucket's attribution, and save it as part of bucket info, which end user app developer could have some use for it. My understanding was either bucket is attributed or not( boolean state). If I use it as a string, what info I can save in it?

Reply:
I'm not understanding why you need attribution on the bucket

Reply:
removed it .. upon discussion